packages/sit-onyx/src/components/OnyxInput/types.ts

```diff
@@ -81,6 +81,11 @@ export type OnyxInputProps = {
    * Message / help text to display below the input.
    */
   message?: string;
+  /**
+   * If `true`, the label will be visually hidden.
+   * For accessibility / screen readers, the aria-label will still be set.
+   */
+  hideLabel?: boolean;
```

Comment:
Please also extend the description for the `label` property to mention this behaviour also.